src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs

```diff
@@ -440,6 +522,26 @@ private class AssemblyMetadata : PackageMetadata
             }
         }
 
+        /// <summary>
+        /// Represents metadata for a Reference item that we want to pretend was resolved as a
+        /// standard NuGet package assembly.
+        /// </summary>
+        private class PreResolvedNetStandardAssemblyMetadata : PackageMetadata
+        {
+            public PreResolvedNetStandardAssemblyMetadata(string name, string path)
+                : base(
+                      type: DependencyType.Assembly,
+                      dependencies: null,
+                      isTopLevelDependency: false,
+                      name: name,
+                      version: string.Empty,
+                      path: path,
+                      resolved: true,
+                      isImplicitlyDefined: false)
```

Comment:
Good question! I think the intent is that references that are not directly in the project file are implicit. Looking through the dependency tree code, I think the relevant code block is [DependencyModel.cs:L41-44](https://github.com/dotnet/project-system/blob/b82639ebeff7912b950ac6e85deaf99069eb27dd/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Models/DependencyModel.cs#L41-L44): implicit nodes do not have a "Remove" action in the context menu.

---
In reply to: [122074507](https://github.com/dotnet/sdk/pull/1330#discussion_r122074507) [](ancestors = 122074507)